tst/EnergyPlus/unit/EMSManager.unit.cc

```diff
@@ -826,6 +827,475 @@ TEST_F( EnergyPlusFixture, EMSManager_CheckIfAnyEMS_OutEMS ) {
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 8 ).Operator, FuncRadToDeg );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 8 ).NumOperands, 1 );
+		index = 8 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR8" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 45.0, 0.0000001 ); // RadToDeg(0.7854 rad) = 45 deg
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 9 ).Operator, FuncExp );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 9 ).NumOperands, 1 );
+		index = 9 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR9" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 2.718281828, 0.00000001 ); // e^1 = 2.71828
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 10 ).Operator, FuncLn );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 10 ).NumOperands, 1 );
+		index = 10 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR10" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 0.69314718, 0.00000001 ); // e^1 = 0.693147
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 11 ).Operator, FuncMax );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 11 ).NumOperands, 2 );
+		index = 11 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR11" );
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 1.5 ); // max(1.5, 0.5) = 1.5
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 12 ).Operator, FuncMin );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 12 ).NumOperands, 2 );
+		index = 12 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR12" );
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 0.5 ); // min(1.5, 0.5) = 0.5
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 13 ).Operator, FuncABS );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 13 ).NumOperands, 1 );
+		index = 13 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR13" );
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 1.3 ); // abs(1.3) = 1.3
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 14 ).Operator, FuncRandU );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 14 ).NumOperands, 2 );
+		index = 14 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR14" );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 15 ).Operator, FuncRandG );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 15 ).NumOperands, 4 );
+		index = 15 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR15" );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 16 ).Operator, FuncRandSeed );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 16 ).NumOperands, 1 );
+		index = 16 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR16" );
```

Comment:
Fixing the min/max for RandomNormal changes the answer but the result is still static.
 @RandomUniform(0.0, 1.0)=0.7526 (OK, I did get a different result here)
 @RandomNormal (0.0, 1.0, 0.0, 1.0)=1.0
 @RandomSeed(1.0)=1.0

Adjusting min/max again gives a different static result:
 @RandomNormal (0.0, 1.0, -10.0, 10.0)=0.0